Dockerfile

```diff
@@ -28,6 +28,27 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Used for platform-specific instructions
 ARG TARGETPLATFORM
 
+# cvxopt, an Augur dependency, does not have pre-built binaries for linux/arm64,
+# so it will have to be built on that platform.
+
+# Add system deps for building cvxopt
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+  apt-get install -y --no-install-recommends \
+      libblas-dev \
+      liblapack-dev \
```

Comment:
libblas-dev and liblapack-dev are the unoptimized reference implementations from https://netlib.org. We probably want to stick with OpenBLAS (libopenblas-dev) or [another optimized implementation of them](https://netlib.org/blas/faq.html#_5_a_id_are_optimized_blas_libraries_available_where_can_i_find_vendor_supplied_blas_a_are_optimized_blas_libraries_available_where_can_i_find_optimized_blas_libraries). OpenBLAS is what cvxopt compiles its own wheels with.